examples/formbot/data/stories.md

```diff
@@ -0,0 +1,29 @@
+## happy path
+* request_restaurant
+    - restaurant_form
+    - form_activated{"form_name": "restaurant_form"}
```

Comment:
random thought, would 
```markdown
- form{"form_name": "restaurant_form"}
<!-- ... -->
- form{"form_name": null}
```
work as well?

Reply:
I think so

Reply:
yeah i can change it to be that i think, if we'd rather that than `form:`

Reply:
no this is not about the form prefix for ignored events but the events to activate / deactivate a form

Reply:
good idea, so we'd have only one event instead of two

Reply:
done